City-of-Pori/pori-kada
# PORI-169 Changed menu to use jquery hover and added hoverIntent

To test:

1. drush cc all
2. Try the desktop menu hover and make sure that there is hoverIntent (https://briancherne.github.io/jquery-hoverIntent/) so that the menu becomes slightly more usable.
